x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/analytics_layout.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { useValues, useActions } from 'kea';
+
+import { KibanaLogic } from '../../../shared/kibana';
+import { FlashMessages } from '../../../shared/flash_messages';
+import { Loading } from '../../../shared/loading';
+
+import { LogRetentionCallout, LogRetentionOptions } from '../log_retention';
+
+import { AnalyticsLogic } from './';
+import { AnalyticsHeader, AnalyticsUnavailable } from './components';
+
+interface Props {
+  title: string;
+}
+export const AnalyticsLayout: React.FC<Props> = ({ title, children }) => {
+  const { history } = useValues(KibanaLogic);
+  const { query } = useParams() as { query: string };
+  const { dataLoading, analyticsUnavailable } = useValues(AnalyticsLogic);
+  const { loadAnalyticsData, loadQueryData } = useActions(AnalyticsLogic);
+
+  useEffect(() => {
+    if (query) {
+      loadQueryData(query);
```

Comment:
Noting here while I review:

> Confirm it correctly fwds search query params

On the initial load (i.e., no filters have been selected yet), there are no query parameters sent on the API request.

It seems like `loadQueryData` simply forward query string parameters on the API request. So if they are not present, they will not be sent.

I have one concern, let me know if it is unfounded.

There are no query string parameters sent on the API request, but there IS a default date range selected in the UI, which I think is 6 days.

Is there also a 6 day default on the server that makes this work correctly? 

If that is the case, are you concerned at all that if the 6 day default ever changed on the server to, let's say, 7 days, that our date box is showing a 6 day time frame but we're actually showing a 7 day time frame?

Let me know if I'm not being clear.

Reply:
Ahh you're totally right, I did change how this worked from ent-search. The standalone UI is always inheriting `startDate` and `endDate` from the server rather than settings its own state/default. I think I changed this because the components changed and I needed a way to control state from the component.

I could actually stand to use a rubber ducky on this, would you mind brainstorming w/ me on this before or during the sync today?

Reply:
Yes, for sure. That would help me as well, because I wasn't actually sure off the top of my head what to suggest as a fix for this.

Reply:
Per our sync convo: 16e9791